openadapt/models.py

```diff
@@ -526,7 +539,16 @@ def get_active_window_event(
         Returns:
             (WindowEvent) the active window event.
         """
-        return WindowEvent(**window.get_active_window_data(include_window_data))
+        window_event_data = window.get_active_window_data(include_window_data)
+        window_event = WindowEvent(**window_event_data)
+
+        if include_window_data:
+            a11y_event_data = window_event_data["state"].copy()
+            a11y_event = A11yEvent(data=a11y_event_data)
+            window_event.a11y_events.append(a11y_event)
+            window_event_data["state"].pop("data")
```

Comment:
I think we want to remove WindowEvent.state

Reply:
done in [8dabc63](https://github.com/OpenAdaptAI/OpenAdapt/pull/842/commits/8dabc634da9fe79fcee89d4bd9eed64bd81613b4)